sdk/python/feast/infra/feature_servers/multicloud/Dockerfile

```diff
@@ -8,9 +8,8 @@ RUN apt update && \
         build-essential
 
 RUN pip install pip --upgrade
-COPY . .
+RUN pip install "feast[aws,gcp,snowflake,redis,go,mysql]"
```

Comment:
Hmmm won't this install the library from pypi, as opposed to a locally built wheel? Can that cause inconsistencies?

Reply:
fixed by restoring `COPY . .`